api/lib/mno_enterprise/concerns/controllers/jpi/v1/teams_controller.rb

```diff
@@ -80,7 +89,14 @@ def update_members(action)
     if params[:team] && params[:team][:users]
       id_list = params[:team][:users].map { |h| h[:id] }.compact
       users = @team.organization.users.where('id.in' => id_list)
-      users.each { |u| @team.send(action, u) }
+
+      user_list = users.map do |u|
+        @team.send(action, u)
+        u.email
+      end.join(',')
+
+      MnoEnterprise::EventLogger.info('team_update', current_user.id, 'Team composition updated', @team.organization,
```

Comment:
```ruby
users.each { |u| @team.send(action, u) }

MnoEnterprise::EventLogger.info('team_update', current_user.id, 'Team composition updated', @team, {action: action.to_s,  users: users.map(&:email)
```

Or you could keep the `map` as you had, it's sligthly more efficent but I think it's harder to read? No strong opinion on this one